component/otelcol/receiver/otlp/otlp.go

```diff
@@ -84,19 +110,43 @@ var (
 		// We almost write 0 bytes, so no need to tune WriteBufferSize.
 	}
 
-	DefaultHTTPServerArguments = HTTPServerArguments{
-		Endpoint: "0.0.0.0:4318",
+	DefaultHTTPServerSettings = HTTPConfigArguments{
+		HTTPServerArguments: &otelcol.HTTPServerArguments{
+			Endpoint: "0.0.0.0:4318",
+		},
+		MetricsURLPath: "/v1/metrics",
+		LogsURLPath:    "/v1/logs",
+		TracesURLPath:  "/v1/traces",
 	}
 )
 
+// Validate implements river.Validator.
+func (args *Arguments) Validate() error {
```

Comment:
I tried and it panics inside of the HTTP pkg in server.go:

```
// Handle registers the handler for the given pattern.
// If a handler already exists for pattern, Handle panics.
func (mux *ServeMux) Handle(pattern string, handler Handler) {
	mux.mu.Lock()
	defer mux.mu.Unlock()

	if pattern == "" {
		panic("http: invalid pattern")
	}
```
I will add a step in the validate to avoid this. good catch